lib/configuration.ts

```diff
@@ -283,6 +284,13 @@ export interface AutomationServerOptions extends AutomationOptions {
          * at the time of this writing.
          */
         port?: number;
+
+        /**
+         * adapterConfig. If not set, uses the default adapter which
+         * adapts to hot-shots statsd at the time of this writing.
+         */
+        adapterConfig?: StatsdAdapterConfig
```

Comment:
I'd like to propose we rename this to `StatsClientFactory` and make it available in the configuration at `statsd.client.factory` to make it similar to other factories that create client implementations like http or ws. 

WDYT?

Reply:
Sounds good to me.